scripts/ci/in_container/entrypoint_ci.sh

```diff
@@ -134,10 +123,6 @@ if [[ ${INTEGRATION_KERBEROS:="false"} == "true" ]]; then
 fi
 
 
-# Start MiniCluster
-java -cp "/opt/minicluster-1.1-SNAPSHOT/*" com.ing.minicluster.MiniCluster \
```

Comment:
Was this left over from Jame's removal of hive? Was it just erroring, but because of the `&` not stopping execution?

Reply:
Exactly.